From 3dae598aa36b0c7250d3ab357f5c6440f1bafe95 Mon Sep 17 00:00:00 2001
From: Meron Cohen <mecohen@habana.ai>
Date: Thu, 08 May 2025 19:06:53 +0300
Subject: [PATCH] [SW-227605] g2e: support G2E variant

In hl-smi, add HL225E revision id (rev 05) and GAUDI2E device type.

Change-Id: I3f37eddd10c6b6bec176638a7482770e5e5e00d2
Signed-off-by: Meron Cohen <mecohen@habana.ai>
---

diff --git a/src/gaudi2.c b/src/gaudi2.c
index c6e2c96..2e2ee80 100755
--- a/src/gaudi2.c
+++ b/src/gaudi2.c
@@ -59,6 +59,7 @@
 #define HL225B_PCIE_REVISION_ID		0x2
 #define HL225C_PCIE_REVISION_ID		0x3
 #define HL225D_PCIE_REVISION_ID		0x4
+#define HL225E_PCIE_REVISION_ID		0x5
 
 static struct hl_smi_clk_info gaudi2_clk[GAUDI2_CLK_NUM];
 static struct hl_smi_clk_limit_info gaudi2_clk_limit[GAUDI2_CLK_LIMIT_NUM];
@@ -1009,6 +1010,8 @@
 		return "C";
 	case HL225D_PCIE_REVISION_ID:
 		return "D";
+	case HL225E_PCIE_REVISION_ID:
+		return "E";
 	default:
 		return "";
 	}
diff --git a/src/utils.c b/src/utils.c
index dbd5cc8..da4a49e 100644
--- a/src/utils.c
+++ b/src/utils.c
@@ -734,6 +734,7 @@
 	"GAUDI2B",
 	"GAUDI2C",
 	"GAUDI2D",
+	"GAUDI2E",
 	"GAUDI2 FPGA",
 	"GAUDI3",
 	"GAUDI3D", /* temporary, LDK preset 328 as G3D */
